build_tools/update_environments_and_lock_files.py

```diff
@@ -108,6 +108,13 @@ def remove_from(alist, to_remove):
         "conda_dependencies": common_dependencies + ["ccache"],
         "package_constraints": {
             "blas": "[build=mkl]",
+            # 2022-06-09 currently mamba install 1.23 and scipy 1.7 which
```

Comment:
Is this issue related to mamba? If not, could we use "conda" here instead?

```suggestion
            # 2022-06-09 currently conda install 1.23 and scipy 1.7 which
```

Reply:
I tweaked the comment to talk about available version in conda defaults channel

Reply:
Actually, I am also confused. Our CI relies on conda-lock to resolve the dependencies which probably uses conda under the hood. Why mention mamba in the comment if this is the case?